README.md

```diff
@@ -3,6 +3,15 @@ W3C Web of Things implementation on NodeJS
 
 [![Build Status](https://travis-ci.org/eclipse/thingweb.node-wot.svg?branch=master)](https://travis-ci.org/eclipse/thingweb.node-wot)
 
+### Table of Contents
+- [Licence](#licence)
+- [Prerequisites](#prerequisites)
+- [How to get the library](#howtoget)
+- [Start with an example](#example)
+- [How to use the library](#api)
+
+<a name="license"/>
```

Comment:
All the anchors, also the others are not needed and I would not add them because they are already there and automatically updated (see https://github.com/eclipse/thingweb.node-wot#prerequisites etc)